spec/smarty_streets_spec.rb

```diff
@@ -1,69 +1,91 @@
 require 'spec_helper'
+require 'net/http'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.